packages/utils/src/lib/log-results.unit.test.ts

```diff
@@ -64,31 +69,27 @@ describe('logPromiseResults', () => {
         } as PromiseFulfilledResult<FileResult>,
       ],
       'Uploaded reports successfully:',
-      result => {
-        console.info(result.value);
-      },
-    );
-
-    expect(console.info).toHaveBeenNthCalledWith(
-      1,
-      'Uploaded reports successfully:',
+      (result): string => result.value.toString(),
     );
-    expect(console.info).toHaveBeenNthCalledWith(2, ['out.json']);
+    const logs = ui()
+      .logger.getRenderer()
+      .getLogs()
+      .map(({ message }) => message);
+    expect(logs[0]).toBe('[ green(success) ] Uploaded reports successfully:');
+    expect(logs[1]).toBe('[ green(success) ] out.json');
   });
 
   it('should log on fail', () => {
     logPromiseResults(
       [{ status: 'rejected', reason: 'fail' } as PromiseRejectedResult],
       'Generated reports failed:',
-      result => {
-        console.warn(result.reason);
-      },
-    );
-
-    expect(console.warn).toHaveBeenNthCalledWith(
-      1,
-      'Generated reports failed:',
+      (result: { reason: string }) => result.reason.toString(),
```

Comment:
The `reason` is already casted as a string.
```suggestion
      (result: { reason: string }) => result.reason,
```